src/main/resources/application.properties

```diff
@@ -62,4 +62,11 @@ cdn.minio.fa.url=
 #Azure Storage
 spring.azure.blob.store.account.name=
 spring.azure.blob.store.account.key=
-spring.azure.blob.store.container.name=
\ No newline at end of file
+spring.azure.blob.store.container.name=
+
+# Email Config
+spring.mail.host=${EMAIL_HOST:#{""}}
+spring.mail.port=${EMAIL_PORT:#{"587"}}
+spring.mail.username=${EMAIL_USERNAME:#{""}}
+spring.mail.password=${EMAIL_PASSWORD:#{""}}
+spring.mail.recipient=${RECIPIENT_EMAILS:#{""}}
```

Comment:
Hey, @pankajjangid05 where are these environment variables being used in this service?